includes/frontend/class-wcmp-cart-fees.php

```diff
@@ -134,7 +135,7 @@ public function get_shipping_tax_class()
         $location = WC_Tax::get_tax_location('');
 
         if (sizeof($location) === 4) {
-            list($country, $state, $postcode, $city) = $location;
+            [$country, $state, $postcode, $city] = $location;
```

Comment:
country is unused, volgens mij kan je hem zo schrijven:
```suggestion
            [, $state, $postcode, $city] = $location;
```